src/baseline_system/parsers/props_wrapper.py

```diff
@@ -188,6 +192,15 @@ def create_implicit_proposition(self, *arg_symbols):
                 "Arguments": arg_symbols
             }
 
+    # retrieve spacy's named entities, only those with relevant label
+    def _get_named_entities(self):
+        relevant_labels = ["PERSON", "NORP", "FACILITY", "ORG", "GPE", "LOC",
+                           "PRODUCT", "EVENT", "WORK_OF_ART", "LANGUAGE",
+                           "DATE", "TIME", "QUANTITY", "MONEY"]
+        doc = self.spacy(unicode(self.sentence))
```

Comment:
I think it would be better to parse with spacy outside of this function, and store it in a class member (```self.doc```).
That way, future functions can access it without requiring to parse again.

Reply:
and also parse to spacy right after we parse with Berkeley

Reply:
accepted. changed it.